src/repairnator-pipeline/src/test/java/fr/inria/spirals/repairnator/pipeline/TestPipeline.java

```diff
@@ -72,7 +72,7 @@ public void testPipelineOnlyGitRepository() throws Exception {
     public void testPipelineGitRepositoryAndBranch() throws Exception {
         GitRepositoryLauncher l = new GitRepositoryLauncher(new String[]{
                 "--gitrepo",
-                "--gitrepourl", "https://github.com/surli/failingProject",
```

Comment:
What about using https://github.com/repairnator/failingProject/ instead? (sent you an invitation)

Reply:
Done!